runtime/moonbase/src/xcm_config.rs

```diff
@@ -493,8 +492,6 @@ pub enum CurrencyId {
 	SelfReserve,
 	// Assets representing other chains native tokens
 	ForeignAsset(AssetId),
-	// Our local assets
-	DeprecatedLocalAssetReserve(AssetId),
```

Comment:
Please keep a dummy variant here, otherwise you will break the Erc20 variant scale index, and that will break dapps

Reply:
Are we encoding/decoding this type? I looked for it and could not find any location where we encode/decode it and that was the reason I decided to remove the variant.

Will have another look at it

Reply:
My bad, it seem's that the only place CurrencyId is scale decoded is via the extrinsics of the xTokens pallet, so actually it will not impact dapps